report/src/problem_analysis/hci/prototyping.tex

```diff
@@ -0,0 +1,94 @@
+Lofi prototypes are typically limited in their function and are used to emphasize concepts within the design, alternative design solutions, and model the general interaction with the artifact or system\cite{low-vs-high-fidelity-prototype}.
+These prototypes are helpful when performing ideation\cite{nielsen-norman-ideation} or when establishing user capabilities in terms of interaction with the system \cite{usefullness-of-different-prototypes,low-vs-high-fidelity-prototype}.
+Lofi prototypes should be used early in the design process to establish the overall direction for a product design. 
+Therefore, the creation of these prototypes and the following evaluations using them should not be a time-consuming or expensive process. \cite{usefullness-of-different-prototypes,low-vs-high-fidelity-prototype} 
+Lofi prototypes are not ideal when evaluating user interaction as they are used to present a design idea to the user, and thus are not interacted with by the user\cite{low-vs-high-fidelity-prototype}.
+
+Unlike lofi prototypes, hifi prototypes provide near-complete functionality and interactivity with a design closely representing the final product.
+They enable users to interact with buttons, icons, and input fields.
+However, due to their detailed and interactive nature, constructing hifi prototypes is more time-consuming than creating lofi prototypes. 
+This makes hifi prototypes more suitable for examining users' interactions with the designed artifact, than exploring new design ideas. \cite{nielsen-norman-prototype-low-vs-high,low-vs-high-fidelity-prototype}
+
+Since the stakeholder has already suggested ideas for a user interface design based on existing systems, we already have a direction for an initial product design. 
+As we want to find a common denominator for the stakeholders' problems described in Section \ref{sec:understanding_the_problem} we need to understand how users interact with a product design resembling a potential solution.
+To this end, a hifi prototype is more suitable than a lofi prototype \cite{low-vs-high-fidelity-prototype}.
+
+\subsection{Developing prototypes}
+When designing a product multiple approaches can be taken.  
+UCD aims to increase effectiveness, accessibility, and sustainability, as well as increase the pleasure the user experiences when using the system\cite{user-centred-design}. 
+An alternative to UCD is LeanUX \cite{Lean_UX}.
+Lean UX is also a user centered approach for UX development that is often used in start-ups.
+The approach focusses on creating minimal viable products(MVPs) that can be iteratively released and tested with representative users.
+
+In this project, the stakeholder has suggested that we based our design on existing solutions.
+Therefore, we endeavor towards a design strategy where we base the design on the current state-of-the-art systems for meeting-management and preferences of the stakeholders in the form of the user stories in Appendix \ref{appendix:user_stories}.
```

Comment:
Have we examined any?

And why introduce both UCD and LeanUX when this description only barely seems to take from UCD?

Reply:
We need to show that we have at least considered which method to choose. We have written just a little bit extra text, hopefully making it more obvious.